medusa/show_queue.py

```diff
@@ -437,36 +438,48 @@ def run(self):
                 return
 
         # TODO: Add more specific indexer exceptions, that should provide the user with some accurate feedback.
-        except IndexerShowIncomplete as e:
+        except IndexerShowNotFound as error:
+            logger.log(u'{id}: Unable to look up show in {path} using id {id} on {indexer}. '
+                       u'Delete metadata files from the folder and try adding it again.'.format(
+                           id=self.indexer_id, path=self.showDir, indexer=indexerApi(self.indexer).name),
+                       logger.WARNING)
+            ui.notifications.error(
+                u'Unable to add show',
+                u'Unable to look up the show in {0} on {1} using ID {2}. '
+                u'Delete metadata files from the folder and try adding it again.'.
+                format(self.showDir, indexerApi(self.indexer).name, self.indexer_id)
```

Comment:
I really dislike having `format` on a new line here. 😕

Reply:
I was following the style of the other exceptions, but...
Oh man, I can't look at this file without wanting to change everything (update the logging and the coding style)

Reply:
True, it's just too much.

Reply:
I'll open a new PR with these style changes.